spec/pragma.dd

```diff
@@ -152,7 +152,7 @@ $(H3 $(LNAME2 crtctor, $(D pragma crt_constructor)))
 
 $(H3 $(LNAME2 crtdtor, $(D pragma crt_destructor)))
 
-    $(P `pragma(crt_destructor)` works the same as `pragma(crt_destructor)` except:)
+    $(P `pragma(crt_destructor)` works the same as `pragma(crt_constructor)` except:)
```

Comment:
Can you merge this section with the following paragraph, I think it's a big ugly. Might as well improve it while we're here.

Reply:
What do you mean, join `pragma crt_constructor` and `pragma crt_destructor` altogether?

Reply:
See how the numbering is done in 9.1.1.2 of the spec.
https://dlang.org/spec/pragma.html#crtctor

Reply:
Yes, I thought about nesting it to make it clearer, although I'm not getting the point of @maxhaton , can you elaborate a bit more? Do you mean I have to duplicate the points where X and Y have in common?

Reply:
> Yes, I thought about nesting it to make it clearer, although I'm not getting the point of @maxhaton , can you elaborate a bit more? Do you mean I have to duplicate the points where X and Y have in common?

Join the two paragraphs together instead of separate numbered points is my interpretation.

Reply:
Joined using `OL` and `LI` macros.